src/chunk.c

```diff
@@ -215,6 +215,52 @@ do_dimension_alignment(ChunkScanCtx *scanctx, Chunk *chunk)
 	return true;
 }
 
+/*
+ * Calculate, and potentially set, a new chunk interval for an open dimension.
+ */
+static bool
+calculate_and_set_new_chunk_interval(Hypertable *ht, Point *p)
+{
+	Hyperspace *hs = ht->space;
+	Dimension  *dim = NULL;
+	Datum		datum;
+	int64		chunk_interval,
+				coord;
+	int			i;
+
+	if (!OidIsValid(ht->chunk_sizing_func) ||
+		ht->fd.chunk_target_size <= 0)
+		return false;
+
+	/* Find first open dimension */
+	for (i = 0; i < hs->num_dimensions; i++)
+	{
+		dim = &hs->dimensions[i];
+
+		if (IS_OPEN_DIMENSION(dim))
+			break;
+
+		dim = NULL;
+	}
+
+	/* Nothing to do if no open dimension */
+	if (NULL == dim)
```

Comment:
Should this be a WARNING? Adaptation on but no open dimensions?

Reply:
This is deep enough in chunk_create that we probably shouldn't kill the transaction here, I think the actual error belongs in the setting.